BetterBioReactor/CyBioReactorMini.cs

```diff
@@ -73,7 +76,7 @@ public void Start()
             int totalContainerSpaces = BioReactor.container.sizeX * BioReactor.container.sizeY;
             numberOfContainerSlots = totalContainerSpaces;
 
-            RestoreItemsFromSaveData();
+            mono.StartCoroutine(RestoreItemsFromSaveData());
```

Comment:
Did you make a dummy monobehaviour just to run a coroutine instead of using the games CoroutineHost?